tests-system/rbt-quantification-object/foo.rsl

```diff
@@ -0,0 +1,9 @@
+package Foo
+
+type Potato {
+    a String
+}
+
+checks Potato {
+    (forall char in a => char != "a"), warning "a should not contain the letter a"
```

Comment:
Please also add the following test:
` (forall char in b => ...`
where `b` does not exist in `Potato`.